mods/e2140/content/shared/sequences/projectiles.yaml

```
@@ -59,10 +54,22 @@ missile_trail:
 projectile_plasma:
 	Defaults:
 		Filename: projectile_plasma.vspr
-	idle:
 		ZOffset: 512
+	idle:
 	explode:
 		BlendMode: Additive
 		Start: 1
 		Length: 5
+
+projectile_plasma2:
```

It's not projectile for plasma cannon, it's ion cannon. So I guess it should be:

```suggestion
projectile_ion:
```